src/drivers/imx/ipc.c

```diff
@@ -77,7 +79,9 @@ static void irq_handler(void *arg)
 		imx_mu_write(IMX_MU_xSR_GIPn(IMX_MU_VERSION, 0),
 			     IMX_MU_xSR(IMX_MU_VERSION, IMX_MU_GSR));
 
+#ifndef CONFIG_ARM64
 		interrupt_clear(PLATFORM_IPC_INTERRUPT);
+#endif /* CONFIG_ARM64 */
```

Comment:
Maybe create a separate empty macro `interrupt_clear` for ARM64 to avoid polluting the code with ifdefs?